aries-site/src/examples/components/layer/LayerStickyScrollExample.js

```diff
@@ -0,0 +1,53 @@
+import React, { useContext, useState } from 'react';
+import {
+  Button,
+  Box,
+  Header,
+  Heading,
+  Layer,
+  ResponsiveContext,
+} from 'grommet';
+import { MonitorFormExample } from './MonitorFormExample';
+
+export const LayerStickyScrollExample = () => {
+  const [open, setOpen] = useState(false);
+  const size = useContext(ResponsiveContext);
+  const onOpen = () => setOpen(true);
+  const onClose = () => setOpen(undefined);
+
+  return (
+    <>
+      <Box align="start">
+        <Button label="Show me the Layer" onClick={onOpen} primary />
+      </Box>
+      {open && (
+        <Layer
+          position="right"
+          full={!['xsmall', 'small'].includes(size) ? 'vertical' : true}
+          modal
+          onClickOutside={onClose}
+          onEsc={onClose}
+        >
+          <Box
+            gap="small"
+            fill="vertical"
+            overflow="auto"
+            width={!['xsmall', 'small'].includes(size) ? 'medium' : undefined}
+            pad="medium"
```

Comment:
Think we need to remove the pad here and place the pad on the form itself and the header itself. Right now we have this issue on scroll:
<img width="393" alt="Screen Shot 2022-04-20 at 10 39 31 AM" src="https://user-images.githubusercontent.com/12522275/164290594-76d64850-217a-45bc-8935-adfa9cddf78d.png">

Reply:
should be fixed now